src/amo/sagas/collections.js

```diff
@@ -0,0 +1,48 @@
+import { all, call, put, select, takeEvery } from 'redux-saga/effects';
+import {
+  FETCH_COLLECTION,
+  loadCollection,
+} from 'amo/reducers/collections';
+import * as api from 'amo/api/collections';
+import log from 'core/logger';
+import { createErrorHandler, getState } from 'core/sagas/utils';
+
+export function* fetchCollection({
+  payload: {
+    errorHandlerId,
+    page,
+    slug,
+    user,
+  },
+}) {
+  const errorHandler = createErrorHandler(errorHandlerId);
+
+  yield put(errorHandler.createClearingAction());
+
+  try {
+    const state = yield select(getState);
```

Comment:
Does this need `yield put(showLoading())` ?

Reply:
Ah yes, maybe indeed.

Reply:
I did not add this as discussed on IRC.